control-userlayer/src/main/java/org/oskari/control/userlayer/CreateUserLayerHandler.java

```diff
@@ -87,7 +93,9 @@ public class CreateUserLayerHandler extends RestActionHandler {
 
     private static final int MAX_RETRY_RANDOM_UUID = 100;
 
-    private final DiskFileItemFactory diskFileItemFactory = new DiskFileItemFactory(MAX_SIZE_MEMORY, null);
+    FileCleaningTracker cleaningTracker = new FileCleaningTracker();
```

Comment:
There's no reference to this? Should it be registered to cleanup temp files somehere?

Reply:
it seems its job is already done on line 175, lets remove this